packages/dcos-integration-test/extra/test_networking.py

```diff
@@ -20,6 +33,36 @@ def _ensure_routable():
     return _ensure_routable
 
 
+@retrying.retry(wait_fixed=2000,
```

Comment:
Also, while you are at it replace the variable name i in the first to to `master` and the second for loop to `slave`.